doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -301,8 +301,20 @@ set with::
 SPDK Usage
 ==================
 
-If you want to use SPDK driver for NVME SSD, you need to specify NVMe serial
-number here with "spdk:" prefix for ``bluestore_block_path``.
+If you want to use SPDK driver for NVME SSD, you need to ready your system.
+Please refer to `SPDK document`__ for more detailes.
+
+.. __: http://www.spdk.io/doc/getting_started.html#getting_started_examples
```

Comment:
please put
```
.. SPDK document: http://www.spdk.io/doc/getting_started.html#getting_started_examples
```

instead.

Reply:
@tchaikov, Kefu, I passed the build_doc check with the patch. When I changed to ".. SPDK document:",  build_doc reported one error.  Thanks!